feathr_project/feathr/client.py

```diff
@@ -325,7 +328,9 @@ def multi_get_online_features(self, feature_table, keys, feature_names):
 
         Args:
             feature_table: the name of the feature table.
-            keys: list of keys for the entities
+            keys: list of keys/key lists for the entities;
```

Comment:
Reader may get confused with the statement:  `the list of keys / key lists`. 
Will be better if we could add more explanation of `composite keys`, at least in docs.

Reply:
Changed the comment to 'list of keys / composite keys' to make it more clear